shared/src/main/java/gov/hhs/cdc/trustedintermediary/external/inmemory/LoggingMetricMetaData.java

```diff
@@ -0,0 +1,44 @@
+package gov.hhs.cdc.trustedintermediary.external.inmemory;
+
+import gov.hhs.cdc.trustedintermediary.metadata.MetaDataEntry;
+import gov.hhs.cdc.trustedintermediary.metadata.MetaDataStep;
+import gov.hhs.cdc.trustedintermediary.wrappers.Logger;
+import gov.hhs.cdc.trustedintermediary.wrappers.MetricMetaData;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import javax.inject.Inject;
+
+/**
+ * Implementation of a class that can be used throughout the project to collect a list of metadata
+ * events
+ */
+public class LoggingMetricMetaData implements MetricMetaData {
+    private static final LoggingMetricMetaData INSTANCE = new LoggingMetricMetaData();
+
+    public static LoggingMetricMetaData getInstance() {
+        return INSTANCE;
+    }
+
+    @Inject Logger logger;
+
+    private LoggingMetricMetaData() {}
+
+    private static final Map<String, Object> metadataMap = new ConcurrentHashMap<>();
```

Comment:
This field...  I just realized that this doesn't make much sense anymore.  Do we need this field?  Can we just instead make a `Map` on the fly in the `put` method?  E.g. `Map.of("BundleId", entry.bundleId(), "Entry Time", entry.entryTime(), "Entry Step", entry.entryStep())`.  Then we can delete this field.  Thoughts?

Reply:
Done.